lib/swaggerUtil.js

```diff
@@ -103,16 +97,16 @@ function validateObject(specs, obj, disc) {
     var validationErrors = [];
     if (!obj[disc]) {//discriminator not defined for this object
         validationErrors.push(
-                {
-                    message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
-                }
+            {
+                message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
```

Comment:
stringifying the object into the message doesn't seem right

if it's worth including, it should go as a second property, e.g.:
```javascript
{
  message: 'discriminator ' + disc + ' not defined for inheriting model',
  model: obj
}
```